src/DynamoManipulation/NodeManipulator.cs

```diff
@@ -181,15 +181,33 @@ protected virtual void MouseUp(object sender, MouseButtonEventArgs e)
         /// <param name="mouseEventArgs"></param>
         protected virtual void MouseMove(object sender, MouseEventArgs mouseEventArgs)
         {
-            if (!CanMoveGizmo(GizmoInAction))
-                return;
-
             var clickRay = BackgroundPreviewViewModel.GetClickRay(mouseEventArgs);
             if (clickRay == null) return;
 
-            var offset = GizmoInAction.GetOffset(clickRay.GetOriginPoint(), clickRay.GetDirectionVector());
-            if (offset.Length < 0.01)
+            if (GizmoInAction == null)
+            {
+                // Check for mouse over highlights on gizmo
+                var gizmos = GetGizmos(false);
+                foreach (var item in gizmos)
+                {
+                    // Delete all transient geometry used to highlight gizmo
+                    item.UnhighlightDrawablesForMouseOver(BackgroundPreviewViewModel);
```

Comment:
So Unhighlight is handled by Gizmo itself whereas highlight is not handled by Gizmo. I think we should keep the API uniform.